src/confdb/gui/JavaCodeExecution.java

```diff
@@ -151,6 +152,33 @@ private void runChecker() {
 		}
 	}
 
+        // CMSHLT-2913: splitting of Datasets and streams for 2023 HIon menu
+        private void customiseForCMSHLT2913() {
+          addSplitStreams("[customiseForCMSHLT2913]", "hltEventContentAForHIRawPrime", "PhysicsHIPhysicsRawPrime", 1, 31);
+          addSplitStreams("[customiseForCMSHLT2913]", "hltEventContentAForHI", "PhysicsHIForward", 1, 2);
+          addSplitStreams("[customiseForCMSHLT2913]", "hltEventContentAForHI", "PhysicsHIMinimumBias", 1, 3);
```

Comment:
Hmm, didn't we agree some time ago that split stuff gets enumerated from 0? IE, 0-31 for 32 fold splitting?

Reply:
Sorry, confusing. Yes, that (0 to N) is what we have in the menu. Here I had to only _add_ the new ones (the 0th one I "created" by hand by renaming `PhysicsHIMinimumBias` to `PhysicsHIMinimumBias0`, etc).

Reply:
Ah OK!